profiles/schema.py

```diff
@@ -50,10 +56,60 @@ def resolve_type(self, info, **kwargs):
     )
 
 
+class ProfilesConnection(graphene.Connection):
+    class Meta:
+        abstract = True
+
+    count = graphene.Int()
+    total_count = graphene.Int()
+
+    def resolve_count(self, info):
+        return self.length
+
+    def resolve_total_count(self, info):
+        return self.iterable.model.objects.filter(
+            service__service_type=SERVICE_TYPES[1][0]
```

Comment:
I would put this in a util function, so that if one day those indices change, we want have to look for them everywhere. maybe `services/utils.py`: 
```python
get_berth_service():
    return SERVICE_TYPES[1][0]
```

you can then just call this function whenever you need.

Reply:
There is actually a task to use real enums with these (OM-155). I think that should fix it.

Reply:
ok, great!